### DISCUSSION TITLE
Errore da qualche giorno

### DISCUSSION THREAD

**Discussion by qeimada** [fattureincloud/api#184] [Q&A] [answered]
Senza aver cambiato una riga di codice, da qualche giorno, se tento di creare una fattura ottengo l'errore:
Exception when calling IssuedDocumentsApi->createIssuedDocument: [404] Client error: `POST https://api-v2.fattureincloud.it/c/******/issued_documents` resulted in a `404 Not Found` response: {"error":{"message":"Cliente non trovato."}}

Da cosa può essere dovuto?

**Comment by valmoz** [comment, marked as answer]
Buongiorno @qeimada,
le suggerisco di leggere la seguente discussione: https://github.com/fattureincloud/api/discussions/178

Grazie
Mauro

**Reply by qeimada** [reply]
Perfetto grazie.
